buddyboss/buddyboss-platform
# PROD-778 #2696-fix-Title showing wrong if Rank Math SEO activated

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2696  .

### How to test the changes in this Pull Request:

1. Activate the Rank Math SEO plugin.
2. Go to the frontend user connection, feed, groups tab, and check the title.

### Proof Screenshots or Video

https://www.loom.com/share/025582bf6454454792932454b54970ae

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
